tools/treasure-trails/index.md

```diff
@@ -206,111 +243,121 @@ include_custom_script:
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20246" title="Please wait ..."><img id="20246-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20246.png" alt="alt text" onclick="javascript:update_rewards('20246','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20251" title="Please wait ..."><img id="20251-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20251.png" alt="alt text" onclick="javascript:update_rewards('20251','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20254" title="Please wait ..."><img id="20254-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20254.png" alt="alt text" onclick="javascript:update_rewards('20254','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20257" title="Please wait ..."><img id="20257-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20257.png" alt="alt text" onclick="javascript:update_rewards('20257','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20260" title="Please wait ..."><img id="20260-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20260.png" alt="alt text" onclick="javascript:update_rewards('20260','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="20263" title="Please wait ..."><img id="20263-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/20263.png" alt="alt text" onclick="javascript:update_rewards('20263','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23392" title="Please wait ..."><img id="23392-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23392.png" alt="alt text" onclick="javascript:update_rewards('23392','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23395" title="Please wait ..."><img id="23395-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23395.png" alt="alt text" onclick="javascript:update_rewards('23395','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23398" title="Please wait ..."><img id="23398-img" class="tt-notselected" src="https://www.osrsbox.com/osrsbox-db/items-icons/23398.png" alt="alt text" onclick="javascript:update_rewards('23398','medium')"></span></div>
+                <div class="col py-1"><span class="osrstooltip" data-type="short" id="23401" title="Please wait ..."><img id="23401-img" class="tt-notselected" src="https://www.o
```

Comment:
Just one small note: the `Clueless scroll` was already in the Medium clue scroll list - so adding it here has made a duplicate in the list. The original one is between the `Gnomish firelighter` and the `Crier hat`. Happy to accept the PR as-is and remove the line myself, or if you want to push the changed to this PR? Just let me know.

Reply:
Interesting, I even Ctrl + F'd for it to see if it was there and I was just missing it. I'll go ahead and remove the duplicate and move the other 3rd age items to the shared section later tonight.

Reply:
Awesome - many thanks! Just did a double check and it is in the `index.html` page twice (found using `20249` item ID).... but it is not in the `rewards.js` file twice. Thanks very much for this PR - it is very much appreciated :)